README.md

```diff
@@ -124,14 +142,13 @@ end
 
 ## Documentation
 
-1. [Installation](./docs/installation.md)
```

Comment:
As I said, I don't want to have the installation instructions in the main readme. I assume that Rubyist kinda knows how to install gems ☺️ This dedicated document was here exactly for that purpose.